src/cascadia/TerminalApp/ActionPreviewHandlers.cpp

```diff
@@ -140,6 +140,33 @@ namespace winrt::TerminalApp::implementation
         });
     }
 
+    void TerminalPage::_PreviewAction(const Settings::Model::ActionAndArgs& args)

```

Comment:
Does it make sense to call this function from `_PreviewActionHandler` to deduplicate the code?

Reply:
derp that was the whole point 🤦